webapp/client/src/components/FormFieldEuroInput.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { IMaskInput } from "react-imask";
+import classNames from "classnames";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabel from "./FieldLabel";
+import FormFieldSeparatedField from "./FormFieldSeparatedField";
+
+function FormFieldEuroInput({
+  fieldName,
+  fieldId,
+  value,
+  required,
+  autofocus,
+  label,
+  currency,
+  fieldWidth,
+  maxLength,
+  details,
+  errors,
+}) {
+  const maxTemplate = "9.99";
+
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+      }}
+      labelComponent={<FieldLabel {...{ label, fieldId, details }} />}
+      render={() => (
+        <div
+          className={classNames(
+            "input-group",
+            "euro-field",
+            `input-width-${fieldWidth}`
+          )}
+        >
+          <IMaskInput
+            id={fieldId}
+            name={fieldName}
+            defaultValue={value}
+            scale={2}
+            type="text"
+            inputMode="numeric"
+            pattern="[0-9]*"
+            mask={Number}
+            padFractionalZeros
+            thousandsSeparator="."
+            radix=","
+            mapToRadix={[]} // Overwrite to prevent that the thousandsSeparator '.' is mapped to radix
+            // This will convert the max length to a max number (prevent max character length and decimal / thousands separator)
+            max={maxTemplate.padStart(maxLength + maxTemplate.length - 1, "9")}
```

Comment:
```suggestion
            max={maxTemplate.padStart(maxLength + maxTemplate.length - 2, "9")}
```
Couldn't we do it like that? Or do you think we should rather change what the backend expects?